lerna.json

```diff
@@ -13,6 +13,7 @@
     "modules/demo/ipc/*",
     "modules/demo/ssr/*",
     "modules/demo/tfjs/*",
-    "modules/demo/client-server"
+    "modules/demo/client-server",
+    "modules/demo/blazingcluster-server"
```

Comment:
slight nit, but what about making a `modules/demo/blazingsql` directory, so that way we have a place to put more BSQL demos?

Reply:
sure I'm assuming similar to `deck` or like `ipc`